### PR TITLE
Disable subscriptions rejected by push endpoint

### PR DESCRIPTION
**Pull request by manti-by**
Description https://github.com/OpenUpSA/muni-portal-backend/issues/53

Note: Result object won't have status_code and data only if we catch an unhandled exception from `pywebpush`.

### CODE REVIEW

File: muni_portal/core/signals.py [hunk 1]
```diff
@@ -26,15 +27,26 @@ def queue_send_webpush_notification(notification_id):
                 vapid_claims={"sub": f"mailto:{settings.DEFAULT_FROM_EMAIL}"}
             )
             result.status_code = response.status_code
+            result.data = response.json() if response.json() else {}
```
Comment (jbothma) [inline muni_portal/core/signals.py:30]: None is way better as "no value" than empty object, isn't it?

```suggestion
            result.data = response.json() if response.json() else None
```

File: muni_portal/tests/unit/test_webpush.py [hunk 2]
```diff
@@ -11,6 +11,17 @@
 faker = Faker()
 
 
+class Response:
+    def __init__(self, status_code=200, data=None):
+        self.status_code = status_code
+        self.reason = status_code
+        self.text = status_code
+        self.data = data if data else {}
```
Comment (jbothma) [inline muni_portal/tests/unit/test_webpush.py:19]: ditto

File: muni_portal/core/signals.py (hunk 1 above)
Comment (jbothma) [reply muni_portal/core/signals.py:30]: or actually just set it to response.json() - why did you use {}?

Comment (manti-by) [reply muni_portal/core/signals.py:30]: To be sure that we updated value during request. If the field is None/Null we can't say for sure that we did this.

**Pull request merged**